packages/mobile/src/components/audio/AudioPlayer.tsx

```diff
@@ -178,7 +178,8 @@ export const AudioPlayer = () => {
   const playbackRate = useSelector(getPlaybackRate)
   const currentUserId = useSelector(getUserId)
   const uid = useSelector(getUid)
-  const playerBehavior = useSelector(getPlayerBehavior)
+  const playerBehavior =
+    useSelector(getPlayerBehavior) || PlayerBehavior.FULL_OR_PREVIEW
```

Comment:
What about doing this in the selector or slice? Fixes both of the cases we've seen and future instances of the issue

Reply:
Ah, just saw your note in the PR. I see no reason we can't do it in the selector or slice. I don't think the cases where we return null are relevant

Reply:
I considered that (see description). If we are ok with always defaulting to FULL_OR_PREVIEW I prefer that too

Reply:
Yeah, FULL_OR_PREVIEW is and should be the default behavior